examples/MODULE.bazel

```diff
@@ -5,6 +5,26 @@ bazel_dep(name = "rules_zig", version = "20240912.0-41bfe84")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "zml", version = "0.1.0")
 bazel_dep(name = "aspect_bazel_lib", version = "2.8.1.1")
+bazel_dep(name = "rules_python", version = "0.36.0")
```

Comment:
I know this where I started it, but not sure `examples/` is the right place to put this tests.
maybe a top level folder `testing` ? not sure

Reply:
Can do!